packages/api/src/libs/queries/lsp.ts

```diff
@@ -87,15 +95,52 @@ export default (appState: Dependencies) => {
     const result = await getTotalTvlSample(currency);
     return result.value;
   }
+  // tvm calculations, these are just based on tvl
+  async function sumTvm(addresses: string[], currency: CurrencySymbol = "usd") {
```

Comment:
the following functions are almost identical to the previous tvl calculations. basically just helpers to sum tvm, or get various types of data